### DISCUSSION TITLE
goal pictograms

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1466] [Q&A]
I am using open-sdg to create the platform for Ukraine. 

We have the goal names, which are slightly different from the global ones in English. Also we would like to have a Ukraininan version of the site with pictograms in Ukrainian.

Could you, please, help:
(1) where to upload the pictograms in English 
(2) where to upload the pictograms in Ukrainian

**Comment by ukraine-sdg** [comment]
and also, if the pictogram files should have a specific names for English and Ukrainian versions of the site

**Comment by MoBosse** [comment]
Hey @ukraine-sdg If I am not mistaken you have to create a new folder called `goals` in your site repo inside the [assets/img](https://github.com/ukraine-sdg/site/tree/develop/assets/img) folder. In there you have to create one folder for each language (en & uk in your case). 
site/assets/img/goals/en
site/assets/img/goals/uk

This is where the goal icons go (saved as 1.png, 2.png, ...17.png).

**Comment by brockfanning** [comment]
A couple of points to add on to the answer by MoBosse:

1. If possible I recommend you create the images as .svg files, with white foreground and transparent background. The reason that I recommend this is that you will not have to create separate high-contrast versions.
2. After placing the files as described by MoBosse, you will need to tell Open SDG where they are. This is done by setting the `goal_image_prefix`. This is done by the following steps:
    1. Go to your site. For example in your case: https://ukraine-sdg.github.io/site/
    2. Copy the URL of the site
    3. In the footer, click on "Configuration"
    4. Scroll down to "Goal image base URL". Paste in the copied URL, plus `/assets/img/goals`. For example in your case: `https://ukraine-sdg.github.io/site/assets/img/goals`
    5. If you created svg images instead of png, under "Goal image extension" type `svg`. Otherwise you can leave this blank (it defaults to png).
    6. At the top, click "Download configuration" to save the configuration file.
    7. Click "Go to repository"
    8. Upload the configuration file you downloaded above
